common/artifact_spawns/wc_artifact_spawns.txt

```diff
@@ -829,4 +829,23 @@ gorehowl = {
 	artifacts = {
 		gorehowl = { value = 1 }
 	}
+}
+ironfoe = {
+	max_amount = 1
+	spawn_date = -1.1.1
+
+	spawn_chance = {
+		value = 100
+	}
+
+	weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 5816 							# Reginald's greatgrandfather
```

Comment:
Ironfoe doesn't spawn because he's dead.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/76257885-f898af00-626b-11ea-9735-f2582d3228b8.png)

</details>

```suggestion
			dynasty=413 #Windsor
			dynasty_head = { character = PREV }
```
Something like this should work: gives Ironfoe to Windsor's dynasty's head.

Reply:
It's related to [@geffe1's comment](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1286#issuecomment-596790114)

Reply:
fixed, works.